packages/common/src/dto-types.ts

```diff
@@ -76,20 +76,34 @@ export class UpdatePlanDto {
 }
 
 export class CreateStudentDto {
```

Comment:
if this is only used in the Signup api, should change to something like SignUpDto (instead of create-student)